plugins/kubernetes/app/models/kubernetes/release.rb

```diff
@@ -46,8 +46,7 @@ def clients
 
     def pod_selector(deploy_group)
       {
-        release_id: id,
-        deploy_group_id: deploy_group.id,
```

Comment:
should either not pass in deploy_group and make sure all callers work with this ... or still pass it in ...